IPython/core/profileapp.py

```diff
@@ -264,8 +264,6 @@ def init_config_files(self):
             'IPython.kernel.zmq.kernelapp.IPKernelApp',
             'IPython.terminal.console.app.ZMQTerminalIPythonApp',
             'IPython.qt.console.qtconsoleapp.IPythonQtConsoleApp',
-            'IPython.html.notebookapp.NotebookApp',
-            'IPython.nbconvert.nbconvertapp.NbConvertApp',
```

Comment:
Do you want to try with `jupyter_notebook` instead of `IPython.html.notebook.app` and `jupyter_nbconvert` ?

Though it is true that profile will go away, so we might as well remove.